ui/app/components/secure-variable-form.hbs

```diff
@@ -33,17 +33,31 @@
         .
       </p>
     {{/if}}
+    <SecureVariableForm::NamespaceFilter 
```

Comment:
@philrenaud moved data loading logic into this component. Here's the [commit](https://github.com/hashicorp/nomad/pull/13629/commits/cef152780fcfe4b255a360ef16cbe765bb3fa716) so you can review this piece in isolation.